lib/renderer/api/remote.js

```diff
@@ -9,14 +9,30 @@ const {ipcRenderer, isPromise, CallbacksRegistry} = require('electron')
 const resolvePromise = Promise.resolve.bind(Promise)
 
 const callbacksRegistry = new CallbacksRegistry()
-
 const remoteObjectCache = v8Util.createIDWeakMap()
 
+// lookup for ipc renderer send calls
+const ipcs = {
```

Comment:
I think this is making the code more complicated and harder to understand due to the extra indirection and the unintuitive abbreviations. What would you think about reverting this bit?

Reply:
hmm, my original thinking was that it would be easier to coalesce these strings into a centralized lookup but i definitely agree that the abbreviations are pretty unintuitive. do you think there might be a better way to group them together, or should i just revert it entirely?

Reply:
I think it would be probably be better revert it for now and leave possible improvements for another PR/issue.